Libraries/Components/View/ReactNativeStyleAttributes.js

```diff
@@ -47,4 +48,16 @@ ReactNativeStyleAttributes.shadowColor = colorAttributes;
 ReactNativeStyleAttributes.textDecorationColor = colorAttributes;
 ReactNativeStyleAttributes.tintColor = colorAttributes;
 
+if (Platform.OS === 'android') {
+  // ReactAndroid doesn't currently have the capability to have variable typing
+  // (eg. string or array) of a native attribute.  To work around that, we can
+  // transform variable types into one standard type, as done here with
+  // textDecorationLine which can be an array of enum values or a space-separated
+  // string containing the same values.
+  var processTextDecorationLine = require('processTextDecorationLine');
```

Comment:
Good catch; this actually didn't work, but I hadn't gone back to test iOS after implementing Android.  I added a stub processTextDecorationLine.ios.js in the PR update that throws new Error('not required on iOS'), but it will be packaged on iOS despite not being required.